src/plugins/lua/luaplugin.cpp

```diff
@@ -44,15 +44,41 @@
 //#define POLYGON_FORMAT_PAIRS
 //#define POLYGON_FORMAT_OPTIMAL
 
+namespace {
+
+    void writeColor(Lua::LuaTableWriter &writer,
+                    const char *name,
+                    const QColor &color)
+    {
+        // Example: backgroundcolor = { 255, 200, 100 }
+        writer.writeStartTable(name);
+        writer.setSuppressNewlines(true);
+        writer.writeValue(color.red());
+        writer.writeValue(color.green());
+        writer.writeValue(color.blue());
+        if (color.alpha() != 255)
+            writer.writeValue(color.alpha());
+        writer.writeEndTable();
+        writer.setSuppressNewlines(false);
+    }
```

Comment:
I see no reason for moving this function. Please just keep it where it was as `static`.

Reply:
Done